acquisition_info.html

```diff
@@ -0,0 +1,32 @@
+<html lang="en">
+
+<head>
+    <meta http-equiv="origin-trial"
+        content="AkJLziV412NUQqPW2IAlpix6/iDNrc3aTm4o5gzD9j09pSrTq/cWdUgPiTe1y7aVGrFilozKb/Pbw2Np6IJT0JQAAAB0eyJvcmlnaW4iOiJodHRwczovL3B3YXRlc3RhcHBzLmdpdGh1Yi5pbzo0NDMiLCJpc1N1YmRvbWFpbiI6ZmFsc2UsImZlYXR1cmUiOiJtc0FjcXVpc2l0aW9uSW5mbyIsImV4cGlyeSI6MTcxNjI0OTYwMH0=">
```

Comment:
Do you need to retain a link to the manifest and service worker as well?

Reply:
I'm not super familiar with the process, but my assumption was that the manifest and service worker inclusion would be if we needed offline capability for this.